docs/user/requirements.md

```diff
@@ -1,13 +1,25 @@
 ---
 Title: Requirements
-Description: I can use my 7 years old computer to run the K-Box?
+Description: Hardware and Software prerequisites
```

Comment:
You really want this boring default string to feed the search engine browsing our documentation?

Reply:
:) Probably yes.